src/plugins/filters/component/condition.js

```diff
@@ -98,10 +98,11 @@ class ConditionComponent extends BaseComponent {
 
   /**
    * Update state of component.
-   * @param {Object} condition Object with keys:
-   *  * `command` Object, Command object with condition name as `key` property.
-   *  * `args` Array, Condition arguments.
-   * @param column Physical column index.
+   *
+   * @param {object} condition The condition object.
+   * @param {object} condition.command The command object with condition name as `key` property.
+   * @param {Array} condition.args An array of values to compare.
+   * @param {number} column Physical column index.
```

Comment:
But function has 2 arguments.

Reply:
This actually is a correct description. If the function argument is passed as an object we can document what properties that object contains. Here the docs say that passed the `condition` object must have `command` and `args` keys. Moreover, it is generated correctly https://handsontable.com/docs/7.3.0/ConditionComponent.html#updateState.